lib/src/casting_manager.dart

```diff
@@ -53,6 +55,10 @@ class BitmovinCastManager implements BitmovinCastManagerApi {
   );
 
   @override
-  Future<void> updateContext() =>
-      _mainChannel.invokeMethod(Methods.castManagerUpdateContext);
+  Future<void> updateContext() {
```

Comment:
As this is not supported on iOS, I do not call the method channel here in that case